### PR TITLE
Nuvlabox playbook - opt-in workflow at creation time

### PR DESCRIPTION
**Pull request by cjdcordeiro**
closes https://github.com/nuvla/ui/issues/758

### CODE REVIEW

**Review by 0xbase12 — CHANGES_REQUESTED** [review]

File: code/src/cljs/sixsq/nuvla/ui/edge/events.cljs [hunk 1]
```diff
@@ -387,3 +387,18 @@
...
   (fn [db [_ active-tab-index]]
     (assoc db ::spec/active-tab-index active-tab-index)))
+
+
+(reg-event-fx
+  ::enable-host-level-management
+  (fn [_ [_ nuvlabox-id]]
+    {::cimi-api-fx/operation [nuvlabox-id
+                              "enable-host-level-management"
+                              #(dispatch [::set-nuvlabox-playbooks-cronjob %])
```
[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply code/src/cljs/sixsq/nuvla/ui/edge/events.cljs:397]: error msg added

File: code/src/cljs/sixsq/nuvla/ui/edge/views.cljs [hunk 2]
```diff
@@ -383,11 +411,13 @@
         ; default ttl for API key is 30 days
         default-ttl                30
         usb-trigger-key-ttl        (r/atom default-ttl)
+        playbooks-toggle           (r/atom false)
         ssh-toggle                 (r/atom false)
         ssh-existing-key           (r/atom false)
         ssh-chosen-keys            (r/atom [])
         nuvlabox-ssh-keys          (subscribe [::subs/nuvlabox-ssh-key])
         new-private-ssh-key        (subscribe [::subs/nuvlabox-private-ssh-key])
+        playbooks-cronjob          (subscribe [::subs/nuvlabox-playbooks-cronjob])
```
[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply code/src/cljs/sixsq/nuvla/ui/edge/views.cljs:420]: won't that affect how soon that variable becomes available?
changed anyway

File: code/src/cljs/sixsq/nuvla/ui/edge/views.cljs [hunk 3]
```diff
@@ -193,9 +196,33 @@
...
            [ui/Segment {:basic true}
-            [CreateSSHKeyMessage @new-private-ssh-key private-ssh-key-file tr]])
+            (when playbooks-toggle
+              [ui/Message {:icon true}
+               [ui/Icon {:name (if @playbooks-cronjob "check circle outline" "spinner")
+                         :loading (if @playbooks-cronjob false true)}]
+               [ui/MessageContent
+                [ui/MessageHeader [:span (@tr [:nuvlabox-playbooks-cronjob]) " "
+                                   (when @playbooks-cronjob
+                                     [ui/Popup {:content @playbooks-cronjob
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/edge/views.cljs:208]: Now that I merge this [PR](https://github.com/nuvla/api-server/commit/c69c9e3344aba25eecd27ad599d75379f9b5079c) this PR need to be fixed

**Comment by 0xbase12** [comment]
This pull request can be closed because it's included in the https://github.com/nuvla/ui/pull/766?

**Comment by cjdcordeiro** [comment]
@0xbase12 yes it can

**Pull request closed without merging**